miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/WorkflowStateDto.java

```diff
@@ -33,13 +35,15 @@ public WorkflowStateDto(Workflow workflow) {
       this.message = prompt.getMessage();
       this.inputTypes = prompt.getInputTypes();
     }
+    this.lastModified = workflow.getProgress().getLastModified().toString();
```

Comment:
Ooh, I think this is an opportunity for a bit of cleanup, as we missed this code the first time around... Kashif, I know you weren't the one to write this method, but I'm hoping you can be the one to streamline it! All of our DTOs are supposed to be built in the Dtos class, like you did with the WorkflowNameDto. This WorkflowStateDto constructor (and the one below on line 45) should be moved to the `Dtos` class, so that the Dto itself holds only getters and setters. There's also a `formatDateTime` method in the `Dtos` class that will standardize the conversion of `lastModified`.